zerver/tests/test_openapi.py

```diff
@@ -902,6 +902,7 @@ def test_attributes(self) -> None:
             "scheduled_messages",
             "mobile",
             "invites",
+            "saved_snippets",
```

Comment:
This is I think not necessary, if we're putting them in the `drafts` or `users` section?

Reply:
Added them to `drafts` section.